packages/generate-component-docs/src/__snapshots__/contract.test.ts.snap

```diff
@@ -8923,7 +8923,7 @@ exports[`filterSuggestions 1`] = `
     some: (predicate: (value: FilterableSuggestion<Value> | FilterableGroupedSuggestions<Value>, index: number, array: (FilterableSuggestion<...> | FilterableGroupedSuggestions<...>)[]) => unknown, thisArg?: any) => boolean
     sort: (compareFn?: ((a: FilterableSuggestion<Value> | FilterableGroupedSuggestions<Value>, b: FilterableSuggestion<Value> | FilterableGroupedSuggestions<...>) => number) | undefined) => FilterableSuggestions<...>
     splice: { (start: number, deleteCount?: number | undefined): (FilterableSuggestion<Value> | FilterableGroupedSuggestions<Value>)[]; (start: number, deleteCount: number, ...items: (FilterableSuggestion<...> | FilterableGroupedSuggestions<...>)[]): (FilterableSuggestion<...> | FilterableGroupedSuggestions<...>)[]; }
-    toLocaleString: () => string
+    toLocaleString: { (): string; (locales: string | string[], options?: (NumberFormatOptions & DateTimeFormatOptions) | undefined): string; }
```

Comment:
The type serializer treats exported functions as components, which in this case results in the entire `Array` prototype being serialized. Have seen this occur in metropolis too, could be worth addressing just to prevent unnecessary snapshot churn when new JS APIs get created.